src/player.h

```diff
@@ -166,6 +166,11 @@ class Player
 
 	std::string inventory_formspec;
 
+	bool custom_deathscreen;
+	bool enable_deathscreen;
+	std::string get_deathscreen_formspec() { return deathscreen_formspec; }
+	void set_deathscreen_formspec(std::string input) { deathscreen_formspec = input; }
```

Comment:
we don't pass std::string as copy if they are not modified before doing things. Please use const std::string & instead